nixos/tests/paperless.nix

```diff
@@ -1,32 +1,30 @@
 import ./make-test-python.nix ({ lib, ... }: {
-  name = "paperless-ng";
+  name = "paperless";
   meta.maintainers = with lib.maintainers; [ earvstedt Flakebi ];
 
   nodes.machine = { pkgs, ... }: {
     environment.systemPackages = with pkgs; [ imagemagick jq ];
-    services.paperless-ng = {
+    services.paperless = {
       enable = true;
       passwordFile = builtins.toFile "password" "admin";
     };
   };
 
   testScript = ''
-    import json
+    machine.wait_for_unit("paperless-consumer.service")
 
-    machine.wait_for_unit("paperless-ng-consumer.service")
-
-    with subtest("Add a document via the file system"):
+    with subtest("Create test doc"):
```

Comment:
I think this is a regression from the previous commit?

Reply:
Thanks for catching this. History rewrites across renames can be a bit tricky.